server/issue.go

```diff
@@ -26,7 +26,7 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	api := ji.GetPlugin().API
 
 	create := &struct {
-		RequiredFieldsNotCovered []string         `json:"required_fields_not_covered"`
+		RequiredFieldsNotCovered [][]string       `json:"required_fields_not_covered"`
```

Comment:
Nice - proper and simple fix! Thanks for figuring this out.

Why a slice and not a map?

Reply:
@levb Well, I would have preferred a tuple, but we don't have those. I thought since we're controlling what goes into that slice, it's okay to assume it's a tuple slice. I thought I map would be overkill. But 0/5 on it, your choice.